docs/book/src/plugins/go-v4-plugin.md

```diff
@@ -24,6 +24,8 @@ under the [testdata][testdata] directory on the root directory of the Kubebuilde
 - If you are looking to have your project update with the latest version available
 - if you are not targeting k8s versions < `1.16` and `1.20` if you are using webhooks
 - If you are looking to work on with scaffolds which are compatible with k8s `1.25+`
+- If you are looking for the new layout following the [Standard Go Project Layout][standard-go-project] where the "api(s)" and "controller(s)" are scaffold 
```

Comment:
See : https://github.com/kubernetes-sigs/kubebuilder/pull/2985#issuecomment-1327616257 ( that is not possible )

Reply:
I think I understand now. It is not possible trying to put internal under pkg, but that is not what the standard layout suggests. The internal directory should be at the project level. And I see a lot of projects doing that to avoid exporting everything.

The pkg directory actually is more debatable. See https://eli.thegreenplace.net/2019/simple-go-project-layout-with-modules/ as an example.

But I do not have a strong opinion on this. The current layout works for me also. Bottom line, a controller project should in general not be imported by other projects.